_includes/concepts/unmanaged.html

```diff
@@ -0,0 +1,70 @@
+<h3>Manual Tags</h3>
+<p>These tags are key value pairs used by humans to coordinate activity. Generally, they reflect some static-ish attributes about a system (for example, <code>HAS_GPU=true</code> could be used to signal to systems that an asset contains at least one GPU. More commonly, humans set these tags on assets in a transitive manner, to mark and later query for a set of assets to work on. For example, a user may mark 20 assets with <code>CLAIMED=gabe</code> and later query for assets with <code>CLAIMED=gabe</code> to reprovision these hosts into new roles.</p>
+
+<h3>Automated Tags</h3>
```

Comment:
That should be discussed in deleteSomeMetaOnRepurpose https://tumblr.github.io/collins/configuration.html#features